test/sqatt/src/Sqatt.hs

```diff
@@ -154,6 +159,13 @@ decodePath filePath =
       throwError $ FilePathError $
         "Cannot decode " <> apprPath <> " properly"
 
+-- | Replace the characters that might cause problems on Windows systems.
+toFSSafeStr :: String -> String
+toFSSafeStr str = repl <$> str
```

Comment:
Why use <$>?
I had to look up its meaning (https://stackoverflow.com/questions/37286376/what-does-mean-in-haskell)
Why not `toFSSafeStr = map repl`?

Reply:
I'd like to use the `fmap` function whenever possible since it generalizes `map` to any functor. In this case it does not matter much, but in general I'd prefer to use `fmap` since it provides the freedom to change the functor later in time.